libs/pages/onboarding/ui/src/lib/components/step-pricing/step-pricing.tsx

```diff
@@ -1,10 +1,179 @@
+import { Button, ButtonSize, ButtonType, InputSelect, Value } from "@console/shared-ui"
```

Comment:
We use only a single quote and we didn't use semicolon, please use Prettier on saving, it's very important!